jbpm-services/jbpm-kie-services/src/test/resources/multi-remotable/pojo-pom.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+ <modelVersion>4.0.0</modelVersion>
+
+ <groupId>org.test</groupId>
+ <artifactId>jbpm-kie-services-filter-test-pojo</artifactId>
+ <version>1.0.0-SNAPSHOT</version>
+
+ <dependencies>
+  <dependency>
+   <groupId>org.kie</groupId>
+   <artifactId>kie-api</artifactId>
+   <version>6.4.0.Final</version>
```

Comment:
we need to reference a version of kie-api as it uses @Remotable annotation. I guess we could change it to same version as project just wondering if we use 6.5.1-SNAPSHOT will that be automatically replaced with next version on release time, @psiroky @mbiarnes wdyt?